frontend/src/pages/MyCourses/MyCourses.tsx

```diff
@@ -1,52 +1,72 @@
 import React from 'react';
 
 import { AuthorizedLayout } from 'components/Layout';
-
+import { PageContainer, CourseButton, CourseActions } from './styled';
```

Comment:
`AuthorizedLayout` in `My Courses` as like `Corses` use `Grid` with 2 row?

Reply:
My courses don't use grid

Reply:
Ok, fine